src/app.ts

```diff
@@ -79,205 +44,8 @@ const rateLimiter = rateLimit({
 })
 app.use(rateLimiter)
 
-app.get('/test', (req, res) => {
```

Comment:
I think we could actually use such an endpoint, but maybe name it `/status` and make it return the text `OK`.